commands/utility/get_user_info.js

```diff
@@ -2,11 +2,14 @@ const { SlashCommandBuilder, PermissionFlagsBits } = require('discord.js');
 /*  ----------------------
 Google Sheets Setup
 -------------------------- */
-const { spreadsheetId } = require('../../config.json');
+//const { spreadsheetId } = require('../../config.json');
```

Comment:
is there a reason we're getting rid of the config.json?

Reply:
Github action are unable to read files being gitignored. We passed github secrets as environment variables instead